uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

```diff
@@ -0,0 +1,177 @@
+<div class="modal-header">
+  <button type="button" class="close" ng-click="modal.closeModal()"><span aria-hidden="true">&times;</span><span
+    class="sr-only">Close</span></button>
+  <h4 class="modal-title">Leave Request</h4>
+</div>
+<div class="modal-body clearfix">
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="chr_custom-select chr_custom-select--full">
+        <select>
+          <option value="1" selected>Vacation (15 days available)</option>
+          <option value="2">Medical</option>
+        </select>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-xs-12">
+      <uib-tabset class="chr-tab__outer">
```

Comment:
Why do we need this `.chr_tab__outer`? As far as I understand you're simply overriding the default style of bootstrap tabs, so then why you just don't do that, instead of creating a new component?